src/pages/api/geocode.ts

```diff
@@ -0,0 +1,22 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { withSentry } from '@sentry/nextjs';
+
+async function geocodeHandler(req: NextApiRequest, res: NextApiResponse) {
+    try {
+        const request = await fetch(
+            `https://maps.googleapis.com/maps/api/geocode/json?latlng=${req.query?.latlng}&result_type=locality&key=${process.env.GOOGLE_PLACES_API_KEY}`
```

Comment:
do we want to add a short circuit here to check the value of `req.query`? using optional chaining here might result in unnecessary fetches to `.../geocode/json?latlng=undefined&...`. might be a non-issue if `req.query` is never null